cypress/integration/pages/frontPage/testsForCanonicalOnly.js

```diff
@@ -13,6 +19,23 @@ export const testsThatFollowSmokeTestConfigForCanonicalOnly = ({
     it('should not have an AMP attribute', () => {
       cy.get('html').should('not.have.attr', 'amp');
     });
+    // Once the most read is implemented for amp and all other pages this test will be moved to testforallpages
+    if (serviceHasMostRead(service) && Cypress.env('APP_ENV') !== 'live') {
```

Comment:
Should this also run on the live environment?

Reply:
oh yeah can remove it as this will be merged after the most read goes live.

Reply:
done.